coverage/clover.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need to have the coverage folder pushed to the repo and what would its purpose be??

Reply:
Thanks, @wasibani-roy for the highlight, I have removed it from version control.